lib/index.js

```diff
@@ -121,7 +121,7 @@ const changelog =
 const emptyPlugins = []
 /** @type {Readonly<RemarkRehypeOptions>} */
 const emptyRemarkRehypeOptions = {allowDangerousHtml: true}
-const safeProtocol = /^(https?|ircs?|mailto|xmpp)$/i
+const safeProtocol = /^(https?|ircs?|mailto|xmpp|tel)$/i
```

Comment:
```suggestion
const safeProtocol = /^(https?|ircs?|mailto|tel|xmpp)$/i
```

Let's sort the protocols in the `RegExp`.